### DISCUSSION TITLE
LazyTerm started opening Whichkey when I press space

### DISCUSSION THREAD

**Discussion by guzman109** [LazyVim/LazyVim#4077] [Q&A] [answered]
Does anyone know how to stop it from activating whichkey?

**Comment by dpetka2001** [comment]
https://github.com/folke/which-key.nvim#%EF%B8%8F-configuration You probably want to disable terminal mode I think.

**Reply by guzman109** [reply]
That worked but now there's a delay when pressing space. This just stared happening this morning.

**Reply by dpetka2001** [reply]
You mean delay in `insert` mode when in Terminal? Maybe you have some kind of mapping in terminal mode that starts with `<space>`?

**Reply by folke** [reply]
That;s the only explaination. whichk-key also wouldn't pop up otherwise.
Op probably has a `<leader>` mapping in the terminal

**Reply by guzman109** [reply]
Yes, I toggle lazyterm and when I type a command with a space. For example typing `cd Documents`, there will be a delay when I type the space before Documents.
 
I have no keymaps, its all default. LazyVim was updated this morning when I opened it and it started happening.

**Reply by folke** [reply, marked as answer]
I can't reproduce this at all.
What does `:verbose tmap <leader>` say?

**Reply by guzman109** [reply]
Ah ok, there was one plugin that had a default keymap in terminal mode. I went ahead and disabled it.

**Reply by guzman109** [reply]
Thanks for your help!
